updating/understanding-openshift-updates.adoc

```diff
@@ -13,3 +13,12 @@ The graph is based on recommended, tested update paths from a specific version.
 After the CVO receives the update image from the registry, the CVO then applies the changes.
 
 include::modules/update-common-terms.adoc[leveloffset=+1]
+
+[role="_additional-resources"]
+[discrete]
```

Comment:
So if you want this additional resources section to appear as if it's additional resources for the module, then you'd want to use the `.Additional resources` format (and remove the [discrete] heading).